### ISSUE TITLE
Questions about finetuning. 

### ISSUE THREAD

**Issue by pansanity666** [googleinterns/IBRNet#16]
Hi,  
    I have several confusions about the finetuning for the Generalizable NeRF task (IBRNet, MVSNeRF, etc):
        1. How many frames are used during finetuning? Only the reference views or the whole test set of the new scene?
        2. If the latter, the metrics are calculated with the SEEN frames during finetuning. Is it reasonable? 
        3. Should I lower down the LR during finetuning?

 Best,

**Comment by qianqianwang68** [comment]
Hi, the finetuning only uses the reference views (the test views are kept held out). Yes, you could try lowering the learning rate during fine-tuning and see if that leads to better results.

**Comment by pansanity666** [comment]
> Hi, the finetuning only uses the reference views (the test views are kept held out). Yes, you could try lowering the learning rate during fine-tuning and see if that leads to better results.

Thanks.
